tests/tutor/test_tutor_teacher.py

```diff
@@ -0,0 +1,1330 @@
+"""Test the Tutor teacher functions."""
+
+from tests.markers import nondestructive, test_case, tutor
+
+
+@test_case('')
+@tutor
+def test_edit_an_unopened_hw(tutor_base_url, selenium, teacher):
```

Comment:
If you have the space, go ahead and use 'homework' because that is more descriptive when the test runs.